core/src/main/java/org/mapfish/print/map/image/ImageLayer.java

```diff
@@ -157,30 +169,47 @@ public double getImageBufferScaling() {
   public void prepareRender(
       final MapfishMapContext transformer,
       final MfClientHttpRequestFactory clientHttpRequestFactory) {
-    image = fetchImage(transformer, clientHttpRequestFactory);
+    try {
+      image = fetchImage(transformer, clientHttpRequestFactory);
+    } catch (Exception e) {
+      if (failOnError) {
+        throw new RuntimeException(e);
+      } else {
+        LOGGER.error("Error while fetching image", e);
+        image = createErrorImage(new Rectangle(1, 1));
+        imageLoadError = true;
+        imageBufferScaling = 1;
+        return;
+      }
+    }
+    imageLoadError = false;
+
     final ReferencedEnvelope envelopeOrig =
         transformer.getBounds().toReferencedEnvelope(transformer.getPaintArea());
     final Rectangle paintArea = calculateNewBounds(image, envelopeOrig);
 
+    double widthImageBufferScaling = paintArea.getWidth() / transformer.getMapSize().getWidth();
+    double heightImageBufferScaling = paintArea.getHeight() / transformer.getMapSize().getHeight();
     imageBufferScaling =
-        ((paintArea.getWidth() / transformer.getMapSize().getWidth())
-                + (paintArea.getHeight() / transformer.getMapSize().getHeight()))
-            / 2;
+        Math.sqrt(
+            (Math.pow(widthImageBufferScaling, 2) + Math.pow(heightImageBufferScaling, 2)) / 2);
+    if (imageBufferScaling >= 2) {
```

Comment:
Please choose between removing or adding a config parameter to enable this feature, and include a test to show behaviour.